lib/tasks/cap_cutover.rake

```diff
@@ -478,199 +59,4 @@ namespace :cap_cutover do
     end
     Rails.logger.info total_running_count.to_s + ' in ' + distance_of_time_in_words_to_now(start_time)
   end
-
-  def build_pub_from_sw_and_pubmed(pub)
-    pmid = pub.pmid.to_s
-
-    sciencewire_source_record = SciencewireSourceRecord.where(pmid: pmid).first
-    if sciencewire_source_record.present?
-      sw_pub_hash = sciencewire_source_record.source_as_hash
-      pub.update_attributes(
-        active: true,
-        title: sw_pub_hash[:title],
-        year: sw_pub_hash[:year],
-        sciencewire_id: sw_pub_hash[:sw_id],
-        pub_hash: sw_pub_hash,
-        pages: sw_pub_hash[:pages],
-        issn: sw_pub_hash[:issn],
-        publication_type: sw_pub_hash[:type])
-      pub.add_any_pubmed_data_to_hash
-    else
-      pubmed_source_record = PubmedSourceRecord.where(pmid: pmid).first
-      pubmed_hash = pubmed_source_record.source_as_hash
-      pub.update_attributes(
-        active: true,
-        title: pubmed_hash[:title],
-        year: pubmed_hash[:year],
-        pub_hash: pubmed_hash)
-    end
-    pub.cutover_sync_hash_and_db
-    pub.save
-
-  rescue => e
-    @cap_import_logger.info e.message
-    @cap_import_logger.info e.backtrace.inspect
-    @cap_import_logger.info 'the offending pmid: ' + pmid.to_s
-  end
```

Comment:
Why is `build_pub_from_sw_and_pubmed` removed?  It's part of the analysis in #356.